### PR TITLE
Restricted Instagram, Linkedin and GMB service for basic plan

### PR DESCRIPTION
**Pull request by girishpanchal30**
Close https://github.com/Codeinwp/tweet-old-post-pro/issues/527

### CODE REVIEW

File: includes/admin/class-rop-global-settings.php [hunk 1]
```diff
@@ -621,4 +626,24 @@ public function get_all_services_handle() {
...
+			$product_key  = 'tweet_old_post_pro';
+			$license_data = get_option( $product_key . '_license_data', '' );
+			if ( isset( $license_data->created_at ) ) {
+				$created_at = strtotime( date( 'Y-m-d', strtotime( $license_data->created_at ) ) );
+				$compare_to = strtotime( '2024-07-21' );
+				return $created_at > $compare_to;
+			}
+		}
+
+		return false;
```
Comment (Soare-Robert-Daniel) [inline includes/admin/class-rop-global-settings.php:647]: NIT. Just a personal style, we can make this look nicer:

```php
public function check_is_new_license() {
    if ( $this->license_type() <= 0 ) { // Ignore free users.
        return false;
    }

    $product_key = 'tweet_old_post_pro';
    $license_data = get_option($product_key . '_license_data', '');

    if ( ! isset( $license_data->created_at ) ) {
        return false;
    }

    // Is after 9.1?
    $created_at = strtotime(date('Y-m-d', strtotime($license_data->created_at)));
    $compare_to = strtotime('2024-07-21');

    return $created_at > $compare_to;
}
```

Comment (girishpanchal30) [reply includes/admin/class-rop-global-settings.php]: @Soare-Robert-Daniel Thank you for the review, I've applied the provided code with my latest commit.

**Comment by rodica-andronache** [comment]
@girishpanchal30 when using just the Free version, when you try to add an account for FB or Twitter, the layout of the page changes  ( the buttons from the sidebar, move at the bottom of the page, in the background ). Also, the popup is more shrinked than before.  
I tried to show this in the video, comparing it to the current version of the plugin
https://vertis.d.pr/v/gSHuyV

**Comment by girishpanchal30** [comment]
@rodica-andronache I've tested with local and it seems not to break any UI, Can you please share your test instance details here?
Ref: https://tinyurl.com/263bsuvn

**Comment by rodica-andronache** [comment]
@girishpanchal30 
```
https://exotic-mamba-29f6e6.vertisite.cloud/wp-admin/admin.php?page=TweetOldPost
user: rodica
pass: rodica
```

**Comment by girishpanchal30** [comment]
@rodica-andronache Now I've tested with local and browserstack it seems working properly. Please check the private window and let me know
Ref: https://tinyurl.com/23qslh9s

**Comment by rodica-andronache** [comment]
@girishpanchal30 it's the same for me, in incognito or on browserstack https://www.loom.com/share/3782026dc5a944648c63511e88bab761?sid=60b50ea1-8e44-4bc4-bb95-6c919da75266

**Comment by girishpanchal30** [comment]
@rodica-andronache Fixed, It happened because we have removed the modal footer notice(https://github.com/Codeinwp/tweet-old-post/pull/980) and some CSS issues with the new ROP version.

**Comment by rodica-andronache** [comment]
@girishpanchal30 thank you, now the popup is fine 👍 I'll wait for an answer from @stefan-cotitosu here https://github.com/Codeinwp/tweet-old-post-pro/issues/527#issuecomment-2220424132 to move it to Ready to Merge

**Pull request merged**